docs/read-write-property-behavior.md

```diff
@@ -14,11 +14,28 @@ var fakeShop = A.Fake<ICandyShop>();
 
 fakeShop.Address = "123 Fake Street";
 
-System.Console.Out.Write(fakeShop.Address); 
+System.Console.Out.Write(fakeShop.Address);
 // prints "123 Fake Street"
 ```
 
-This behaviour can be used to 
+This behavior can be used to

```

Comment:
For consistency with the rest of the documents.

We have 11 occurrences of "behaviour" (all in the docs), and 271 occurrences of "behavior" (35 of them in the docs)...